.gitignore

```diff
@@ -0,0 +1,31 @@
+### OSX ###
+.DS_Store
+.AppleDouble
+.LSOverride
+
```

Comment:
Most of this could be covered by a global `.gitignore`, I'd recommend setting one up. https://help.github.com/articles/ignoring-files/#create-a-global-gitignore